src/containers/guest-home-page/login-dialog/LoginDialog.jsx

```diff
@@ -56,7 +56,7 @@ const LoginDialog = () => {
             handleChange={handleInputChange}
             handleSubmit={handleSubmit}
           />
-          <GoogleLogin buttonWidth={styles.form.maxWidth} type={login} />
+          <GoogleLogin buttonWidth={339} type={login} />
```

Comment:
guess, you shouldn`t remove styles.form.maxWidth, just change the width in styles

Reply:
fixed